diff --git a/README.md b/README.md
index d533e32..a0bc122 100644
--- a/README.md
+++ b/README.md
@@ -1,3 +1,3 @@
-python3 src/train.py.py --algo=APPO --env=doom_deadly_corridor --experiment=hallway_exp_gtest --save_every_sec=30 --use_rnn=True --use_geo=True --experiment_summaries_interval=500
+python3 src/train.py --algo=APPO --env=doom_deadly_corridor --experiment=hallway_exp_gtest --save_every_sec=30 --use_rnn=True --use_geo=True --experiment_summaries_interval=500
 
-python src/eval.py.py --algo=APPO --env=doom_deadly_corridor --experiment=hallway_exp_gtest --load_checkpoint_kind=best
\ No newline at end of file
+python src/eval.py --algo=APPO --env=doom_deadly_corridor --experiment=hallway_exp_gtest --load_checkpoint_kind=best
\ No newline at end of file
diff --git a/src/viz_doom_sf/doom_gym.py b/src/viz_doom_sf/doom_gym.py
index b90d91f..289f211 100644
--- a/src/viz_doom_sf/doom_gym.py
+++ b/src/viz_doom_sf/doom_gym.py
@@ -854,7 +854,7 @@ class VizdoomEnvGeo(VizdoomEnv):
                     "prev":0
                 },  
                 "player-inside-sector": {
-                    "data":-np.ones((2,2)),
+                    "data":-np.ones((self.MAX_NODES+1,2)),
                     "graph":{},
                     "prev":0
                 },
@@ -1206,11 +1206,11 @@ class VizdoomEnvGeo(VizdoomEnv):
             for node in self.get_all_nodes_from_graph(["sector","player"]):
 
                 if not node.id in object_ids:
-                    if self.node_in_graph(node.id,node_type):
+                    if self.node_in_graph(node.id,node.type):
                         for connection in self.get_all_node_connections(node.id):
                             self.disconnect_nodes(connection.n1,connection.n2,connection.type)
 
-                        self.delete_node(node.id,node_type)
+                        self.delete_node(node.id,node.type)
 
             for type in self.graph_data["N"]:
                 count = 1
